### DISCUSSION TITLE
How to get the progress of zipping or unzipping single file?

### DISCUSSION THREAD

**Discussion by AAChartModel** [nmoinvaz/minizip#361] [Q&A] [answered]
Is there any method that I can get the progress of processing single file?

**Comment by nmoinvaz** [comment, marked as answer]
There are examples on how to use the library and get progress found in the minizip.c source file. See `minizip_add_progress_cb` and `minizip_extract_progress_cb`. If you want to pull the progress, then don't use the zip readers and writer and you can copy the code out of the zip readers and writers.
